index.html

```diff
@@ -11,6 +11,41 @@
 </head>
     <body>
         <div id = "root"></div>
+        <div id="navigation">
```

Comment:
All of this HTML should be a part of the JavaScript components that we had already made (rather than hard-coded here in `index.html`. Recall the `Header.js` example from earlier.